### ISSUE TITLE
[Visual Requirements - Graph>simple graph]: Shades of color under 'Color dialog' are not visible in high contrast mode.

### ISSUE THREAD

**Issue by natellium** [grafana/grafana#66311]
### User Experience:
Users with low vision who depend on this functionality will not be able to identify the colors and interact with it properly and get disoriented if the color shades are not visible in High contrast mode.

### Pre-requisite: 
Applying High contrast theme.

### Repro steps:
Open #URL: https://play.grafana.org/  Sign in with valid credentials.  "Home" page appears. Navigate the page. Navigate the Feature Showcase pane.  Select any one of the data such as, "Time Series Graphs". "Visualizations" page opens, Navigate to the "Simple Graphs" pane and its description. Navigate to 'Backend_01' legend and select color. Color dialog open. Apply high contract theme. Verify the issue.

### Actual Result
Issue: Shades of color under 'Color dialog' are not visible in high contrast mode.
Observation: Color is the only sole medium to differentiate between backend1 and backend2 graph line. While navigating with screen reader it announces the color shades name i.e. "Dark red, light orange". Expected Result

 Shades of color under 'Color dialog' should be visible in high contrast mode.
 Note:  The same issue exists with the 'Color dialog' throughout the application. Same issue appears for "thresholds" same issue exists for legend->info

### Area
VizLegendListItem Component (Dashboards)

### Severity
`SEV2`

**Comment by evictorero** [comment]
**Context**
High Contrast Mode is a Windows feature, I was able to reproduce it on a Windows machine. Mac OS has a similar feature called Invert colors where this issue is not happening.
For a quick reproduction, there is a Chrome feature under rendering options where is possible to activate `forced-colors` and emulate the same behavior, opening Chrome tools and Rendering options (cmd+shift+P type rendering). 

https://github.com/grafana/grafana/assets/20256983/035f1208-70fa-4b64-9bc5-2b2082283e1f
